samples/IdentityServer.ServerSample/Controllers/AccountController.cs

```diff
@@ -23,8 +24,8 @@ public class AccountController : Controller
 
         public IActionResult Login(string returnUrl)
         {
-            var schemes = _authenticationSchemeProvider.GetAllSchemesAsync();
-            var providers = schemes.Result
+            Task<IEnumerable<AuthenticationScheme>> schemes = _authenticationSchemeProvider.GetAllSchemesAsync();
```

Comment:
Hmm, i had missed this, but why are we not doing async/await?

Reply:
No idea actually! It should be